covizu/utils/batch_utils.py

```diff
@@ -98,7 +95,7 @@ def beadplot_serial(lineage, features, args, callback=None):
         beaddict.update({'sampled_variants': len(labels)})
         beaddict['nodes'].update({variant: []})
 
-        for coldate, accn, label1 in intermed:
+        for coldate, accn, location, label1 in intermed:
```

Comment:
this is a deprecated function - this change was made to allow a unit test to pass